engine/runtime/src/test/scala/org/enso/interpreter/test/instrument/RuntimeVisualizationsTest.scala

```diff
@@ -1877,8 +1877,9 @@ class RuntimeVisualizationsTest
               message =
                 "Method `does_not_exist` of type Main could not be found.",
               stack = Vector(
-                Api.StackTraceElement("<eval>", None, None, None),
-                Api.StackTraceElement("Debug.eval", None, None, None)
+// empty stack for now
```

Comment:
Unfortunate change. Result of trying to unify working with stacktraces behind `InteropLibrary`. Let's see if it can be somewhat mitigated tomorrow.